lib/trie/database_test.go

```diff
@@ -66,6 +67,7 @@ func TestTrie_DatabaseStoreAndLoad(t *testing.T) {
 		err = res.Load(db, trie.MustHash())
 		require.NoError(t, err)
 		require.Equal(t, trie.MustHash(), res.MustHash())
+		require.Equal(t, trie.String(), res.String())
```

Comment:
Maybe then remove the hash assertion above? I guess diffs looks easier with this than comparing the root hashes so this addition makes sense.